project_closing/README.rst

```diff
@@ -20,6 +20,10 @@ To use this module, you need to:
 * Click on *Archive*
 * The associated analytic account will also be archived.
 
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/{repo_id}/{branch}
```

Comment:
You forgot to replace the {repo_id} and {branch} placeholders

Reply:
Oh no... XD